<launch>
  <node pkg="pyopenvr_wrapper_ros" type="service_pose_node.py" name="service_pose_node" cwd="node" output="screen">
  </node>
</launch>
